simulator-core/src/test/java/com/evbox/everon/ocpp/functional/availability/HeartbeatTest.java

```diff
@@ -0,0 +1,55 @@
+package com.evbox.everon.ocpp.functional.availability;
+
+import com.evbox.everon.ocpp.testutils.ocpp.exchange.Heartbeat;
+import com.evbox.everon.ocpp.testutils.station.StationSimulatorSetUp;
+import org.junit.jupiter.api.Test;
+
+import java.time.Instant;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+
+import static com.evbox.everon.ocpp.testutils.constants.StationConstants.STATION_ID;
+import static com.evbox.everon.ocpp.testutils.expect.ExpectedCount.atLeastOnce;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+
+public class HeartbeatTest extends StationSimulatorSetUp {
+
+    private static final ZonedDateTime SERVER_TIME = ZonedDateTime.of(2035, 1, 1, 1, 1, 1, 0, ZoneOffset.UTC);
```

Comment:
Magic constant... why 2035?

Reply:
It can be anything, it was the same timestamp in the shouldAdjustCurrentTimeBasedOnHeartbeatResponse (before refactoring). The test just asserts that we set the station timestamp based on the heartbeat response from the server.